amr-wind/wind_energy/actuator/turbine/fast/FastIface.cpp

```diff
@@ -159,6 +159,12 @@ void FastIface::advance_turbine(const int local_id)
     for (int i = 0; i < fi.num_substeps; ++i, ++fi.time_index) {
         fast_func(FAST_OpFM_Step, &fi.tid_local);
     }
+
+    if ( ( fi.time_index / fi.num_substeps) % fi.chkpt_interval == 0) {
+        char rst_file[fast_strlen()];
+        copy_filename(" ", rst_file);
```

Comment:
```suggestion
        rst_file[0] = '\0';
```

No need to incur a function call here. I think OpenFAST strips the string anyway, but you should confirm that.